plop-templates/Page/Page.tsx.hbs

```diff
@@ -6,7 +6,7 @@ import {
 
 import { PageShell } from '../../recipes/PageShell/PageShell';
 
-export const {{pascalCase name}}: React.FC = () => (
+export const {{pascalCase name}} = () => (
```

Comment:
could you also update `Page.stories.tsx.hbs` to no longer use `Props`, since this file doesn't export any? it's caused a few TS errors -- I think the ProjectOverview file now has an example with `Props` removed